### PR TITLE
Fix $to and $toName type in docblock

### PR DESCRIPTION
**Pull request by RomainMazB**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Just a comment fix to ease IDE analysis
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Doesn't fix anything
| How to test?      | There is nothing more to test
| Possible impacts? | Absolutely not

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23272)

### CODE REVIEW

File: classes/Mail.php [hunk 1]
```diff
@@ -113,8 +113,8 @@ class MailCore extends ObjectModel
      * @param string $template Template: the name of template not be a var but a string !
      * @param string $subject Subject of the email
      * @param array $templateVars Template variables for the email
-     * @param string $to To email
-     * @param string $toName To name
+     * @param string|string[] $to To email
+     * @param string|string[] $toName To name
```
Comment (Progi1984) [inline classes/Mail.php]:
```suggestion
     * @param string|array<string> $to To email
     * @param string|array<string> $toName To name
```

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by Progi1984 — APPROVED** [review]

**Comment by RomainMazB** [comment]
I'm okay with that: I'm not working with PS for so long, dunno yet the code style.

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @RomainMazB

**Pull request merged**